doc/source/reference/ufuncs.rst

```diff
@@ -350,6 +350,28 @@ advanced usage and will not typically be used.
     and for generalized ufuncs for which all outputs are scalars, the output
     tuples can be omitted.
 
+*axis*
+
+    .. versionadded:: 1.15
+
+    A single axis over which a generalized ufunc should operate. This is a
+    short-cut for ufuncs that operate over a single, shared core dimension,
+    equivalent to passing in ``axes`` with entries of ``(axis,)`` for each
+    single-core-dimension argument and ``()`` for all others.  For instance,
+    for a signature ``(i),(i)->()``, it is equivalent to passing in
+    ``axes=[(axis,), (axis,), ()]``.
+
+*keepdims*
+
+    .. versionadded:: 1.15
+
+    If this is set to `True`, an axis which is reduced will be left in the
+    result as a dimension with size one. With this option, the result will
+    correctly against the inputs. This option can only be used for generalized
```

Comment:
Note to myself: missing `broadcast` and this incorrectly states the single dimension needs to be shared.